servobuild.example

```diff
@@ -33,13 +35,15 @@ debug-mozjs = false
 
 # Android information
 [android]
-sdk = "/opt/android-sdk"
-ndk = "/opt/android-ndk"
-toolchain = "/opt/android-toolchain"
+# Defaults sdk = "" ndk = "" toolchain = ""
```

Comment:
Defaults to "unspecified" (and will instead pull from the `ANDROID_SDK`/etc env vars